src/ApiException.php

```diff
@@ -151,13 +256,26 @@ public static function createFromRestApiResponse(
      */
     private static function create($basicMessage, $rpcCode, $metadata, array $decodedMetadata, $previous = null)
     {
+        $containsErrorInfo = self::containsErrorInfo($decodedMetadata);
         $rpcStatus = ApiStatus::statusFromRpcCode($rpcCode);
-        $messageData = [
-            'message' => $basicMessage,
-            'code' => $rpcCode,
-            'status' => $rpcStatus,
-            'details' => $decodedMetadata
-        ];
+        if ($containsErrorInfo['containsErrorInfo'] === true) {
```

Comment:
Don't need to compare to `true`. If the key doesn't exist it will return `null` I believe which is falsey.